projects/fal/tests/test_stability.py

```diff
@@ -11,11 +11,8 @@
 PACKAGE_NAME = "fal"
 
 
+@pytest.mark.xfail(reason="after migrating to cloudpickle we get a serialization error")
 def test_missing_dependencies_nested_server_error(isolated_client):
-    from fal import _serialization
-
-    _serialization._PACKAGES.clear()
-    _serialization._MODULES.clear()
```

Comment:
This is pretty much the only place where we were doing this, so even though we no longer have the mechanism to clear these I don't think we really need it yet. But we would likely do a pickle patch context instead of blasting `patch_pickle()` at module level like we used to do and like we are doing right now.